src/components/price/service.ts

```diff
@@ -59,21 +72,17 @@ export class PriceService {
         try {
             const pairPrice = this.getPairPrice(swap.network, swap.outputNetwork);
 
-            const inputDecimals = Config[swap.network].decimals;
-            const outputDecimals = Config[swap.outputNetwork].decimals;
+            const inputDecimals = this.blockchainConfig[swap.network].decimals;
+            const outputDecimals = this.blockchainConfig[swap.outputNetwork].decimals;
 
             const priceInBig = mulDecimals(pairPrice, outputDecimals);
 
             const outputAmount = mul(priceInBig, divDecimals(swap.inputAmount, inputDecimals));
 
-            const requestedAmount = mul(
-                outputAmount,
-                add(
-                    1,
-                    AppConfig.PRICE.TOLERANCE[swap.network + '-' + swap.outputNetwork] ||
-                        AppConfig.PRICE.TOLERANCE.DEFAULT
-                )
-            );
+            const pairSlippage = this.userConfig.PAIRS[`${swap.network}-${swap.outputNetwork}`].SLIPPAGE || 0;
+
+            const requestedAmount = mul(outputAmount, add(1, pairSlippage));
```

Comment:
Slippage property should be renamed to MIN_MATCHING_FEE or MIN_MATCHING_SPREAD

If it is slippage the logic should be different